streams/src/main/java/org/apache/kafka/streams/processor/Cancellable.java

```diff
@@ -14,10 +14,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.streams.processor.internals;
+package org.apache.kafka.streams.processor;
 
-public interface Punctuator {
+public interface Cancellable {
```

Comment:
They don't need to implement it but they may use it, it's returned from ProcessorContext.schedule so I assumed it shouldn't go to internals package but I may be wrong.